index.html

```diff
@@ -94,27 +90,25 @@
             </li>
           </ul>
           <span class="campaign">
-            <a href="#">
-             CC Campaign
-            </a>
+            <a href="#"> CC Campaign </a>
           </span>
         </nav>
       </section>
-      
-        <section class="banner">
-          <h3>"Welcome to the"</h3>
-          <h1>Web development Hackathon</h1>
-          <p class="banner-description">
-            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Est
-            laboriosam quod minus natus, facilis dicta autem explicabo.
-          </p>
-          <p class="banner-program">2015.10.15(Thu) ~ 16(FRI)</p>
-          <p class="banner-location">
-            @National Museum of Corea, Art center Nabi and more
-          </p>
-        </section>
+
+      <section class="banner">
+        <h3>"Welcome to the"</h3>
+        <h1>Web development Hackathon</h1>
+        <p class="banner-description">
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Est
+          laboriosam quod minus natus, facilis dicta autem explicabo.
+        </p>
+        <p class="banner-program">2015.10.15(Thu) ~ 16(FRI)</p>
+        <p class="banner-location">
+          @National Museum of Corea, Art center Nabi and more

```

Comment:
- [ ] OPTIONAL: Please notice that the content is hardly readable because the image is too bright, I suggest you to make background color darker: 

I suggest  | As it is
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/152655534-9eade841-1036-4615-bc2d-350f5c711f61.png) | ![image](https://user-images.githubusercontent.com/77038610/152655448-468a6f4b-d341-401f-938f-00958ee71e88.png)

Here is how you can make the background `img` darker 😉 : 
`background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)) ,url(https://image..);`

- [ ] Same comment goes to about page

Reply:
Sure I will work on this too